From 722130c19f2ca88f234618380c5fdcc46fd4a3ed Mon Sep 17 00:00:00 2001
From: Andrii Osipov <aosipov@anaconda.com>
Date: Thu, 20 Nov 2025 06:01:37 -0800
Subject: [PATCH] update clean test

---
 tests/test_clean.py | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/tests/test_clean.py b/tests/test_clean.py
index 1d263b3..f534fc5 100644
--- a/tests/test_clean.py
+++ b/tests/test_clean.py
@@ -736,7 +736,12 @@ def test_nonexistent_namespace():
             ),
         ),
         "track",
-        "wbr",
+        pytest.param(
+            "wbr",
+            marks=pytest.mark.xfail(
+                reason="html5lib 1.1 bug - fixed in git but not released. Will pass when html5lib >1.1 is available."
+            ),
+        ),
     ],
 )
 def test_self_closing_tags_self_close(tag):
-- 
2.39.3 (Apple Git-146)

